ui/component/viewers/videoViewer/internal/videojs.jsx

```diff
@@ -299,7 +303,7 @@ export default React.memo<Props>(function VideoJs(props: Props) {
           // (2) We'll have to get 'makeSelectClientSetting(SETTINGS.VIDEO_THEATER_MODE)'
           // as a prop here so we can say "Theater mode|Default mode" instead of
           // "Toggle Theather mode".
-          controlBar.getChild('Button').controlText(__('Toggle Theater mode (t)'));
+          // controlBar.getChild('Button').controlText(__('Toggle Theater mode (t)'));
```

Comment:
Why is this removed?

Reply:
Was causing a clash with the new button, both were saying "Theater mode"